packages/fxa-settings/src/components/PageChangePassword/index.tsx

```diff
@@ -124,142 +131,167 @@ export const PageChangePassword = ({}: RouteComponentProps) => {
   };
 
   return (
-    <FlowContainer title="Change Password">
-      {alertBarRevealed && alertText && (
-        <AlertBar onDismiss={hideAlertBar} type="error">
-          <p data-testid="sign-out-error">Error text TBD. {alertText}</p>
-        </AlertBar>
-      )}
-      <form onSubmit={handleSubmit(onFormSubmit)}>
-        <h1>Stay safe — don't reuse passwords. Your password:</h1>
+    <Localized id="pw-change-header" attrs={{ title: true }}>
+      <FlowContainer title="Change Password">
+        {alertBarRevealed && alertText && (
+          <AlertBar onDismiss={hideAlertBar} type="error">
+            <p data-testid="sign-out-error">Error text TBD. {alertText}</p>
+          </AlertBar>
+        )}
+        <form onSubmit={handleSubmit(onFormSubmit)}>
+          <Localized id="pw-change-stay-safe">
+            <h1>Stay safe — don't reuse passwords. Your password:</h1>
+          </Localized>
+          <ul className="text-grey-400 text-xs m-3 list-inside">
+            <Localized id="pw-change-least-8-chars">
```

Comment:
The `ValidationIcon` is missing by having `Localized` here.  Let's just have `Localized` around the text?